tests/js/spec/views/organizationEvents/index.spec.jsx

```diff
@@ -248,6 +252,8 @@ describe('OrganizationEvents', function() {
   });
 
   it('does not update router when toggling environment selector without changes', async function() {
+    expect(router.push).toHaveBeenCalledTimes(1);
```

Comment:
Why does this get called now?

Reply:
`handleUpdateProjects` is called once on mount in order to update the query params